helper/src/main/java/me/lucko/helper/utils/XMaterial.java

```diff
@@ -0,0 +1,1686 @@
+    WATER_BUCKET(0),
+    WET_SPONGE(1, "SPONGE"),
+    WHEAT(0, "CROPS"),
+    WHEAT_SEEDS(0, "SEEDS"),
+    WHITE_BANNER(15, "BANNER", "STANDING_BANNER"),
+    WHITE_BED(0, "BED", "BED_BLOCK"),
+    WHITE_CARPET(0, "CARPET"),
+    WHITE_CONCRETE(0, "CONCRETE"),
+    WHITE_CONCRETE_POWDER(0, "CONCRETE_POWDER"),
+    WHITE_DYE(15, "1.14", "INK_SACK", "BONE_MEAL"),
+    WHITE_GLAZED_TERRACOTTA(0, "1.12", "HARD_CLAY", "STAINED_CLAY", "WHITE_TERRACOTTA"),
+    WHITE_SHULKER_BOX(0),
+    WHITE_STAINED_GLASS(0, "STAINED_GLASS"),
+    WHITE_STAINED_GLASS_PANE(0, "THIN_GLASS", "STAINED_GLASS_PANE"),
+    WHITE_TERRACOTTA(0, "HARD_CLAY", "TERRACOTTA"),
+    WHITE_TULIP(6, "RED_ROSE"),
+    WHITE_WALL_BANNER(15, "WALL_BANNER"),
+    WHITE_WOOL(0, "WOOL"),
+    WITCH_SPAWN_EGG(66, "MONSTER_EGG"),
+    WITHER_ROSE(0, "1.14", "BLACK_DYE"),
+    WITHER_SKELETON_SKULL(1, "SKULL", "SKULL_ITEM"),
+    WITHER_SKELETON_SPAWN_EGG(5, "MONSTER_EGG"),
+    WITHER_SKELETON_WALL_SKULL(1, "SKULL", "SKULL_ITEM"),
+    WOLF_SPAWN_EGG(95, "MONSTER_EGG"),
+    WOODEN_AXE(0, "WOOD_AXE"),
+    WOODEN_HOE(0, "WOOD_HOE"),
+    WOODEN_PICKAXE(0, "WOOD_PICKAXE"),
+    WOODEN_SHOVEL(0, "WOOD_SPADE"),
+    WOODEN_SWORD(0, "WOOD_SWORD"),
+    WRITABLE_BOOK(0, "BOOK_AND_QUILL"),
+    WRITTEN_BOOK(0),
+    YELLOW_BANNER(11, "BANNER", "STANDING_BANNER"),
+    YELLOW_BED(4, "BED", "BED_BLOCK"),
+    YELLOW_CARPET(4, "CARPET"),
+    YELLOW_CONCRETE(4, "CONCRETE"),
+    YELLOW_CONCRETE_POWDER(4, "CONCRETE_POWDER"),
+    YELLOW_DYE(11, "INK_SACK", "DANDELION_YELLOW"),
+    YELLOW_GLAZED_TERRACOTTA(4, "1.12", "HARD_CLAY", "STAINED_CLAY", "YELLOW_TERRACOTTA"),
+    YELLOW_SHULKER_BOX(0),
+    YELLOW_STAINED_GLASS(4, "STAINED_GLASS"),
+    YELLOW_STAINED_GLASS_PANE(4, "THIN_GLASS", "STAINED_GLASS_PANE"),
+    YELLOW_TERRACOTTA(4, "HARD_CLAY", "STAINED_CLAY"),
+    YELLOW_WALL_BANNER(11, "WALL_BANNER"),
+    YELLOW_WOOL(4, "WOOL"),
+    ZOMBIE_HEAD(2, "SKULL", "SKULL_ITEM"),
+    ZOMBIE_HORSE_SPAWN_EGG(29, "MONSTER_EGG"),
+    ZOMBIE_PIGMAN_SPAWN_EGG(57, "MONSTER_EGG"),
+    ZOMBIE_SPAWN_EGG(54, "MONSTER_EGG"),
+    ZOMBIE_VILLAGER_SPAWN_EGG(27, "MONSTER_EGG"),
+    ZOMBIE_WALL_HEAD(2, "SKULL", "SKULL_ITEM");
+
+
+    /**
+     * A list of material names that can be damaged.<br>
+     * Some names are not complete as this list needs to be<br>
+     * checked with {@link String#contains}.
+     */
+    public static final String[] DAMAGEABLE = {
+            "HELMET", "CHESTPLATE", "LEGGINGS", "BOOTS",
+            "SWORD", "AXE", "PICKAXE", "SHOVEL", "HOE",
+            "ELYTRA", "TRIDENT", "HORSE_ARMOR", "BARDING",
+            "SHEARS", "FLINT_AND_STEEL", "BOW", "FISHING_ROD",
+            "CARROT_ON_A_STICK", "CARROT_STICK"
+    };
+    /**
+     * A list of duplicated items in 1.13 and 1.12 with different purpose.<br>
+     * Values are the new material names.<br>
+     * Duplicates are only checked by keys not values.<br>
+     * Checked with {@link String#equals}
```

Comment:
Additional paragraphs in comments should include an empty line between previous paragraphs.